src/components/Dashboard/ActionRequired.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-expansion-panels v-model="panel">
+    <v-expansion-panel class="action-required-container">
+      <v-card flat class="px-6 py-5">
+        <h3>Action Required</h3>
+        <p class="mt-1">
+          BC Registries is missing information about your business (e.g.. business start date, nature of business,
+          business address etc.). Please contact BC Registries to input any missing business information. Missing
```

Comment:
"to input" doesn't sound right. How about "to enter"? (confirm with UI/UX)

Reply:
<img width="912" alt="Screen Shot 2022-05-26 at 11 14 24 AM" src="https://user-images.githubusercontent.com/55506249/170550394-805de914-02d4-4cb7-929e-5231d69ab5d7.png">